pubspec.lock

```diff
@@ -216,6 +216,11 @@ packages:
     description: flutter
     source: sdk
     version: "0.0.0"
+  flutter_web_plugins:
```

Comment:
[This whole file should probably be `.gitignore`d](https://dart.dev/tools/pub/private-files#the-rules) and not in the repo 🙈